smk/gene_annotation.smk

```diff
@@ -337,21 +337,20 @@ rule gene_annot_kofamscan:
         remote_dir=$(dirname {output})
         time (
             exec_annotation -k {input.ko_list} -p {input.prok_hal} --tmp-dir tmp -f mapper-one-line --cpu {threads} -o kofam_mapper.txt {input.faa}
-            echo -e "#Definitions downloaded\t$(stat -c '%y' {minto_dir}/data/kofam_db/ko_list | cut -d' ' -f 1)\t$(stat -c '%y' {input.module_map} | cut -d' ' -f 1)\t$(stat -c '%y' {input.pathway_map} | cut -d' ' -f 1)" > kofam_processed.txt
+            echo -e "#Definitions downloaded\\t$(stat -c '%y' {input.ko_list} | cut -d' ' -f 1)\\t$(stat -c '%y' {input.module_map} | cut -d' ' -f 1)\\t$(stat -c '%y' {input.pathway_map} | cut -d' ' -f 1)" > kofam_processed.txt
             {script_dir}/kofam_hits.pl --pathway-map {input.pathway_map} --module-map {input.module_map} kofam_mapper.txt >>  kofam_processed.txt
             rsync -a  kofam_processed.txt {output}
         ) >& {log}
         """
 
 rule gene_annot_dbcan:
     input:
-        faa=rules.rename_prokka_sequences.output.faa
+        faa=rules.rename_prokka_sequences.output.faa,
+        dbcan_db="{minto_dir}/data/dbCAN_db/V12/fam-substrate-mapping.tsv"
```

Comment:
Did it compile without error? I think this will complain that `minto_dir` as wildcard cannot be figured out. Either use format like the old params or `f"...{{...}}..."`